cli/cli/gateway.py

```diff
@@ -115,23 +122,39 @@ def print_routes(self) -> None:
         routes: list[Route] = self.get_routes()
         routes.sort(key=lambda r: r.relative_url)
 
-        table = Table("Relative url", "Target", "HTTP methods")
+        table = Table("Relative url", "Target", "HTTP methods", "JWT", "CORS")
         for route in routes:
+            jwt = "On" if route.jwt else "Off"
+            cors = "On" if route.cors else "Off"
             http_methods = " ".join(route.http_methods) if route.http_methods else "All"
-            table.add_row(route.relative_url, route.target, http_methods)
+            table.add_row(route.relative_url, route.target, http_methods, jwt, cors)
 
         console.print(table)
 
     def get_routes(self) -> list[Route]:
         """Get all routes from Kong."""
+        # Get routes
         resp = self._request(method="GET", url=self.routes_url)
         route_data = {r.get("name"): r for r in resp.json().get("data")}
 
+        # Get services
         resp = self._request(method="GET", url=self.services_url)
         service_data = {s.get("name"): s for s in resp.json().get("data")}
 
+        # Get plugins
+        resp = self._request(method="GET", url=self.plugins_url)
+        plugins_json = resp.json().get("data")
+
+        # Work out which plugins apply to which routes
+        route_plugins = defaultdict(list)
+        for p in plugins_json:
+            plugin_route = p.get("route")
+            if plugin_route and plugin_route.get("id"):
+                route_plugins[plugin_route.get("id")].append(p)
```

Comment:
Can use a walrus here ^^

```suggestion
            if plugin_route and plugin_id := plugin_route.get("id"):
                route_plugins[plugin_id].append(p)
```

Reply:
Would need to be:

```
if plugin_route and (plugin_id := plugin_route.get("id")):
    route_plugins[plugin_id].append(p)
```

Which is a bit gross. I think I prefer:

```
# Work out which plugins apply to which routes                  
route_plugins = defaultdict(list)                               
for p in plugins_json:                                          
    plugin_route = p.get("route")                               
    plugin_id = plugin_route.get("id") if plugin_route else None
    if plugin_id:                                               
        route_plugins[plugin_id].append(p)
```